style/ruby/README.md

```diff
@@ -25,6 +24,7 @@ Ruby
 * Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
   calls.
 * Prefer `if` over `unless`.
+* Prefer explicit, early `return`s over `if`-`else`.
```

Comment:
Seeing as there's some disagreement about

> * Prefer explicit, early `return`s over `if`-`else`.

I'll put in a commit to remove it.

I would be satisfied if guard clauses and conditional modifiers are simply _allowed_ by our style guide.